cmd/entrypoint/subcommands/decode_script_test.go

```diff
@@ -84,7 +84,7 @@ func TestDecodeScriptInvalidBase64(t *testing.T) {
 	invalidData := []byte("!")
 	expectedError := base64.CorruptInputError(0)
 
-	src, err := ioutil.TempFile("", "decode-script-test-*")
+	src, err := os.MkdirTemp("", "decode-script-test-*")
```

Comment:
I believe, also according to https://go.dev/doc/go1.16#ioutil:~:text=TempFile%20%3D%3E%20os.CreateTemp, this shall be CreateTemp. That's what's been failing the build test.

Reply:
It was my oversight, thanks